REPOSITORIES.md

```diff
@@ -0,0 +1,36 @@
+## Repositories
+
+This document describes the lifecycle of repositories under the [Notary Project](https://github.com/notaryproject/) organization.
+
+Anyone can submit proposals regarding repositories' lifecycle by opening a GitHub issue in the
+[notaryproject/.github](https://github.com/notaryproject/.github) repository. The [Notary Project governance maintainers](MAINTAINERS) will take into account the community feedback and decide on the proposal.
+
+### Create a new sub-project
+
+A new sub-project can be created as a repository and contributed to the Notary Project organization by opening a GitHub issue to request [Notary Project governance maintainers](MAINTAINERS) to vote in the
+[notaryproject/.github](https://github.com/notaryproject/.github) repository. A new sub-project can be created after having a supermajority (two-thirds) of approval from Notary Project governance maintainers.
+
+If the decision is to add the proposed project, then one of the Notary Project GitHub organization admins will assist the issue opener in creating a repository or transferring an existing repository to the Notary Project organization. Upon creation, the repository must be reviewed to make sure it follows the [Notary Project Governance](GOVERNANCE.md). 
+
+### Lifecycle change 
+
+A repository can be archived if no activity like Pull Requests for a year. The sub-project maintainers of a given repository can propose changing its status by opening a GitHub issue in the [notaryproject/.github](https://github.com/notaryproject/.github) repository. The [Notary Project governance maintainers](MAINTAINERS) will take into account the community feedback and decide on the proposal. The project status should be stated in the README of the repository.
+
+#### Archiviation
+
+Repositories may become stable or stale, showing little to no activity during the time span of a year. They can be proposed for Archiviation by opening a GitHub [issue](https://github.com/notaryproject/.github/issues). The repository-level maintainers should be noticed about this archiviation issue. Once the proposal issue has a supermajority (two-thirds) of approval from Notary Project governance maintainers and there is no response from the repository-level maintainers after 30 days notice, the repository can be archived. [Archived repositories](https://docs.github.com/en/repositories/archiving-a-github-repository/archiving-repositories) will remain inside the Notary Project GitHub organization but will be read-only and will not be maintained. As such, `CODEOWNERS` files contained in archived repositories are not valid.
```

Comment:
Why we need to archive a stable repo? You cannot even open issues in an archived repository. Suggesting removing the word `stable`.

Reply:
Removed "stable" to avoid ambiguity.